src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+@import '/src/styles/blocks/variables';
```

Comment:
It seems you are trying to import variables. However, it's better to import variables in the main SCSS file (usually `main.scss` or `index.scss`). This way, the variables will be available to all other files and you won't have to import them in each file separately.